MdeModulePkg/Universal/SmmCommunicationBufferDxe/SmmCommunicationBufferDxe.c

```diff
@@ -53,6 +59,12 @@ SmmCommunicationBufferEntryPoint (
   //
   DescriptorSize += sizeof (UINT64) - (DescriptorSize % sizeof (UINT64));
 
+  GuidHob = GetFirstGuidHob (&gEdkiiCommunicationBufferGuid);
+  ASSERT (GuidHob != NULL);

```

Comment:
what happen if not found the gEdkiiCommunicationBufferGuid?

it should allow to continue?